src/ApiUtils/include/ApiUtils/GetFunctionTableAddressPrefix.h

```diff
@@ -14,6 +14,9 @@ namespace orbit_api_utils {
 static const std::string kOrbitApiGetFunctionTableAddressPrefix{
     "orbit_api_get_function_table_address_v"};
 
+static const std::string kOrbitApiGetFunctionTableAddressWinPrefix{
```

Comment:
Should that have a comment?

Reply:
Expanded the comment above and added a short one here.